lib/dialects/abstract/query-generator.js

```diff
@@ -1619,9 +1624,16 @@ class QueryGenerator {
       }
     }
 
+    let joinTableName = this.quoteTable(tableRight, asRight);
+
+    // For mysql, embe
```

Comment:
What is `embe`?

Reply:
Sorry that's probably just a type i left in instead of finishing the comment. Can't recall what i wanted to write there after so long.

Reply:
I see. Please just remove the comment then in order not to confuse others :+1: